season_pass/contracts/src/contract.cairo

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: MIT
+// Compatible with OpenZeppelin Contracts for Cairo ^0.17.0
+
+// Eternum Season Pass
+#[starknet::interface]
+trait IRealmMetadataEncoded<TState> {
+    fn get_encoded_metadata(self: @TState, token_id: u16) -> (felt252, felt252, felt252);
+}
+
+#[starknet::contract]
+mod EternumSeasonPass {
```

Comment:
Where are we going to check that the pass has been used? I think for sanity, we should burn the pass if it is used.

Reply:
used passes would be owned by the realms system contract. we transfer the pass to the realms system contract. we could also burn

Reply:
Ok - let's leave that. Then we can track as we go.